components/esm/cellref.cpp

```diff
@@ -168,9 +168,8 @@ void ESM::CellRef::save (ESMWriter &esm, bool wideRefNum, bool inInventory, bool
     if (mChargeInt != -1)
         esm.writeHNT("INTV", mChargeInt);
 
-    if (mGoldValue != 1) {
+    if (mGoldValue != 0)
```

Comment:
Wouldn't it make sense to initialize mGoldValue by 1 instead? And reset it to 1 for anything that has it as 0. That would allow to skip saving this subrecord for anything that isn't a non-1 septim. Manual references are initialized with 1 for example even though they're often not gold. NAM9 subrecord actually has "number of items" semantics in Morrowind but we don't replicate that yet.

It won't break anything either as it stands atm.

Reply:
Done.